Wikipedia/UI-V5/WMFArticleListCollectionViewController.m

```diff
@@ -54,6 +55,7 @@ - (instancetype)initWithCoder:(NSCoder*)aDecoder {
 }
 
 - (void)commonInit {
+    [self observeArticleUpdates];
```

Comment:
note that if you agree with my rationale and decide to move observation in to `willAppear`/`didDisappear` that the unobservation in `dealloc` becomes unnecessary because 1) we only observe if the view appears and 2) disappear methods should always be called before dealloc if the view was shown.